docs/frontend/login.md

```
@@ -75,12 +80,12 @@ Campo: Password
 ⚠️ IMPORTANTE: I punti rimpiazzano i caratteri (per privacy)
 ```
 
-**4. (Opzionale) Mostra password**
+**4. (Opzionale) Ricordami**
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)